sample-global-distro.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Linking Required Peer JavaScript Dependencies</title>
+</head>
+
+<body>
+    <div id="greet"></div>
+
+    <!--- Must reference peer dependencies here first since these are not included in wrapped global package -->
```

Comment:
If the code paths I'm using in my external project do not require a given peer, can I exclude loading it? (ie. immutable)

Reply:
Yes, you can exclude loading it

Reply:
Actually, I think I'm wrong... lemme look again.

Reply:
Took another look and my original answer is correct, however there is something to be very aware of.  Consider line 99 of `string-utils.ts`.  The exported module is an object literal which executes `Humanize.pluralize` which is executed when the file is first downloaded.  We'll need to stop referencing these directly in the object literal definition and instead _always_ define private functions to avoid this issue.